src/views/PlanningView.vue

```diff
@@ -21,7 +31,7 @@ const panelStore = usePanelsStore()
       <div class="font-poppins font-semibold text-xl w-72">
         Planning d’aménagement du réseau trambus
       </div>
-      <div>Timeline</div>
+      <div><Timeline :items="timeLineItems" @selected-date="updateDate" /></div>
```

Comment:
@akhelouat I think we should use the same name for the component here as in the original component. I.e. `UiTimeLine` not `TimeLine`. So it's less confusing. Just my 2 cent